### ISSUE TITLE
module conversion list for SCIRun-Exchange

### ISSUE THREAD

**Issue by jessdtate** [SCIInstitute/SCIRun#1809]
list of modules that need converting from the SCIRun-Exchange repo.

- [ ] CalculateMeshCenter

**Comment by ChantelC** [comment]
ApplyFiltertoFieldData

**Comment by allywarner** [comment]
- [ ] ShowAndEditDipoles

**Comment by karajohnson** [comment]
ClipLatVolByIndicesOrWidget

**Comment by dcwhite** [comment]
**Analysis (2026-06-13)**

SCIRun-Exchange module conversion tracking. Remaining unchecked items: `ClipLatVolByIndicesOrWidget` and `InterfacewithMatlab`.

`InterfacewithMatlab` depends on the MATLAB engine work in #1221 (deferred, blocked by #1295). `ClipLatVolByIndicesOrWidget` should be convertible independently.

Active module conversion work is on branch `module-config-cleanup` (issue #101 / #1746).
